pkg/insights/resyncer.go

```diff
@@ -119,7 +126,68 @@ func meshScoped(t model.ResourceType) bool {
 	return true
 }
 
-func (p *resyncer) resync() error {
+func (p *resyncer) createOrUpdateServiceInsights() error {
```

Comment:
introduce mutex so we are not updating from ticker and events at the same time

Reply:
thanks, fixed